src/Arcus.Messaging.Pumps.ServiceBus/KeyRotation/AzureServiceBusLocation.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using GuardNet;
+
+namespace Arcus.Messaging.Pumps.ServiceBus.KeyRotation
```

Comment:
This is bigger than just rotation as well:
```suggestion
namespace Arcus.Messaging.Pumps.ServiceBus
```

Reply:
Yes, that's true.